common/dependencies.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-    compile 'com.google.re2j:re2j:1.2'
-    compile "com.google.code.gson:gson:2.8.5"
-    compile 'com.squareup.okhttp3:okhttp:3.11.0'
+    implementation 'com.google.re2j:re2j:1.2'
+    implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'com.squareup.okhttp3:okhttp:3.12.0'
```

Comment:
will test

Reply:
just changed versions, but could not yet change the "compile" to " implementation" (and get rid of the warnings in android), as this breaks the desktop build.

```
java.lang.NoClassDefFoundError: com/google/re2j/Pattern
	at com.frostwire.regex.Pattern.buildStandardPattern(Pattern.java:537)
	at com.frostwire.regex.Pattern.<init>(Pattern.java:71)
	at com.frostwire.regex.Pattern.compile(Pattern.java:81)
	at com.frostwire.desktop.DesktopVPNMonitor.<clinit>(DesktopVPNMonitor.java:42)
	at com.frostwire.desktop.DesktopPlatform.<init>(DesktopPlatform.java:34)
	at com.limegroup.gnutella.gui.GUILoader.load(GUILoader.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.limegroup.gnutella.gui.Main.main(Main.java:79)
Caused by: java.lang.ClassNotFoundException: com.google.re2j.Pattern
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 11 more
```
Even building with Gradle 5.0
```
Angels-MBP:desktop gubatron$ gradle -v

------------------------------------------------------------
Gradle 5.0
------------------------------------------------------------

Build time:   2018-11-26 11:48:43 UTC
Revision:     7fc6e5abf2fc5fe0824aec8a0f5462664dbcd987

Kotlin DSL:   1.0.4
Kotlin:       1.3.10
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.13 compiled on July 10 2018
JVM:          1.8.0_60 (Oracle Corporation 25.60-b23)
OS:           Mac OS X 10.14.2 x86_64

```

Reply:
🤔